geode-core/src/main/resources/org/apache/geode/internal/sanctioned-geode-core-serializables.txt

```diff
@@ -387,59 +387,6 @@ org/apache/geode/internal/datasource/TranxPoolCacheImpl,true,3295652525163658888
-org/apache/geode/internal/memcached/Command$25,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$26,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$27,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$28,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$29,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$3,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$30,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$31,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$32,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$33,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$34,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$4,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$5,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$6,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$7,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$8,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Command$9,false,processor:org/apache/geode/internal/memcached/CommandProcessor
-org/apache/geode/internal/memcached/Reply,false
-org/apache/geode/internal/memcached/Reply$1,false
-org/apache/geode/internal/memcached/Reply$2,false
-org/apache/geode/internal/memcached/Reply$3,false
-org/apache/geode/internal/memcached/Reply$4,false
-org/apache/geode/internal/memcached/Reply$5,false
-org/apache/geode/internal/memcached/Reply$6,false
-org/apache/geode/internal/memcached/Reply$7,false
-org/apache/geode/internal/memcached/Reply$8,false
-org/apache/geode/internal/memcached/Reply$9,false
-org/apache/geode/internal/memcached/ResponseStatus,false
-org/apache/geode/internal/memcached/ResponseStatus$1,false
-org/apache/geode/internal/memcached/ResponseStatus$2,false
-org/apache/geode/internal/memcached/ResponseStatus$3,false
-org/apache/geode/internal/memcached/ResponseStatus$4,false
-org/apache/geode/internal/memcached/ResponseStatus$5,false
-org/apache/geode/internal/memcached/ResponseStatus$6,false
-org/apache/geode/internal/memcached/commands/ClientError,true,-2426928000696680541
```

Comment:
I really don't understand how the sanctioned serializables have historically worked, but is this an indicator that they might be expected to show up on the wire still?  Is this potentially an issue for rolling upgrades?  That was vaguely my understanding of what these tests were supposed to warn against.

Reply:
Good point - I'm not clear if these are sent over the wire. But I moved these sanctioned things into the geode-memcached project and added a AnalyzeSerializables* test to that project was well to validate them.